src/docs/upgrading.dox

```diff
@@ -28,6 +28,15 @@ The performance visualization tool \c wtstats has been removed and is
 no longer supported.
 </dd>
 
+<dt>::wiredtiger_open cache configuration changes</dt>
```

Comment:
Should we also say that existing configurations/setups do not need to do anything or worry about it.

Reply:
I think it is important to mention if the change is not backwards compatible, but in this case compatibility is maintained so it is ok to not mention that.